### PR TITLE
Adds crossed

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

We are removing crossed() so to balance it out I am adding crossed plushie

Sprites made by @katebishop49

## Why It's Good For The Game

Happy birthday Crossed

## Changelog Kate Bishop
:cl:
add: Ghost plush.
/:cl:

### CODE REVIEW

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -574,6 +572,12 @@
 	user.dust(just_ash = FALSE, drop_items = TRUE)
 	return MANUAL_SUICIDE
 
+/obj/item/toy/plush/crossed
+	name = "ghost plushie"
+	desc = "It reminds you of someone important, you just can't make out of who."
```
Comment (BrumalLake) [inline code/game/objects/items/plushes.dm]: typo

**Comment by Fronsis** [comment]
Picture or riot

File: code/game/objects/items/plushes.dm (hunk 1 above)
Comment (Archanial) [reply code/game/objects/items/plushes.dm]: I am blind

Comment (Archanial) [reply code/game/objects/items/plushes.dm]:
```suggestion
	desc = "It reminds you of someone important, you just can't make out who."
```

**Comment by Archanial** [comment]
> 
> 
> Picture or riot

Have you tried using diff bot? 
https://camo.githubusercontent.com/a42b44aaa4f04c87296aebe5984118af70ed631163ed5ae6b2ecff5a62335197/68747470733a2f2f6964622e616666656374656461726330372e636f2e756b2f46696c65732f3136323335363932372f343133313336383434302f312f61667465722e676966

**Comment by Wilchenx** [comment]
> > Picture or riot
> 
> Have you tried using diff bot? https://camo.githubusercontent.com/a42b44aaa4f04c87296aebe5984118af70ed631163ed5ae6b2ecff5a62335197/68747470733a2f2f6964622e616666656374656461726330372e636f2e756b2f46696c65732f3136323335363932372f343133313336383434302f312f61667465722e676966

this is like 10 pixels my man

**Comment by Archanial** [comment]
Better find your glasses then. Or just zoom in.

**Review by ike709 — APPROVED** [review]

**Pull request merged**